README.md

```diff
@@ -116,6 +135,102 @@ The `zstd` format has a clear advantage in speed with reasonable compression rat
 and is likely to become the default compression method in a future release.
 Decompression timings are closely matched among `gz`, `xz`, and `zstd` compression methods.
 
+## Update Vendor Module Versions
+
+Optionally, `obs-service-go_modules` will update module versions in `go.mod`,
+recreate `go.sum`, update the `*.changes` file with any change that's missing
+and create a patch that can be applied when building the package. The version
+changes to be applied need to be specified in a YAML file.
+
+### Service Parameters
+
+To do so, specify the name of the YAML file using the service parameter
+`modupdates`. The default is `none` in which case no version update will be
+performed. You can specify the name of the patch file to generate using the
+service parameter `moddiff`. The default is `go-modules.patch`. If multiple
+`.changes` files exist, missing comments will be added to all files. To apply
+this update to a single `.changes` file, you may use the parameter
+`changesfile`. This parameter can be provided multiple times.
+
+```
+<services>
+  <service name="go_modules" mode="disabled">
+    <param name="modupdates">go-module-updates.yaml</param>
+    <param name="moddiff">go-module.patch</param>
+    <param name="changesfile">mypackage.changes</param>
+  </service>
+</services>
+```
+
+### YAML File
+
+The YAML file has the following format:
+
+```
+go_module:
+       module_updates:
+       - uri: <module>@<version>
+         comment: <comment>
+```
+
+The module information is represented as a list of associative arrays where
+each array represents a single change so multiple changes can be applied. The
+value to the `uri:` key in each array represents the module/version
+information that is passed to `go get` verbatim.
+
+The optional `comment:` key represents additional information that will help
+to determine the purpose of the version update which will be used to generate
+an entry for the `.changes` file. If no comment is specified, a default comment
+will be used.
+
+### Example
+
+Using the [apptainer](https://github.com/apptainer/apptainer) HPC container
+platform as an example to demonstrate module version updates the `_service`
+file:
+
+```
+<services>
+  <service name="go_modules" mode="disabled">
+  <param name="modupdates">go-module-updates.yaml</param>
+  <param name="moddiff">go-module.patch</param>
```

Comment:
Why is that line here when it is the default value? Is there some hidden purpose for it?

Reply:
It's an example of / template for the service file. It's not uncommon to use the default value in examples, is it? There's no hidden agenda, no.